app/backend/src/couchers/couchers_select.py

```diff
@@ -29,62 +29,62 @@ def couchers_select(*expr):
 
 
 class CouchersSelect(Select):
-    def filter_by_username_or_email(self, field):
+    def where_valid_username_or_email(self, field):
         if is_valid_username(field):
-            return self.filter(User.username == field)
+            return self.where(User.username == field)
         elif is_valid_email(field):
-            return self.filter(User.email == field)
+            return self.where(User.email == field)
         # no fields match, this will return no rows
-        return self.filter(False)
+        return self.where(False)
 
-    def filter_by_username_or_id(self, field):
+    def where_valid_username_or_id(self, field):
         if is_valid_username(field):
-            return self.filter(User.username == field)
+            return self.where(User.username == field)
         elif is_valid_user_id(field):
-            return self.filter(User.id == field)
+            return self.where(User.id == field)
         # no fields match, this will return no rows
-        return self.filter(False)
+        return self.where(False)
 
-    def filter_users(self, context, table=User):
+    def where_users_visible(self, context, table=User):
         """
         Filters out users that should not be visible: blocked, deleted, or banned
 
         Filters the given table, assuming it's already joined/selected from
         """
         hidden_users = _relevant_user_blocks(context.user_id)
-        return self.filter(table.is_visible).filter(~table.id.in_(hidden_users))
+        return self.where(table.is_visible).where(~table.id.in_(hidden_users))
 
-    def filter_users_column(self, context, column):
+    def where_users_column_visible(self, context, column):
```

Comment:
Looking at how this is used, I think it might make sense to rename this to something like `where_user_visible`, e.g. 
```.where_users_column_visible(context, HostRequest.from_user_id)``` would become ```.where_user_visible(context, HostRequest.from_user_id)```

Reply:
So same method name, different parameters?

Reply:
Nevermind, Python doesn't support overloading. Having two method calls, `select(...).where_user_visible(...)` vs `select(...).where_users_visible(...)` seems quite confusing to me. I'm not sure what a better name is, but this seems a poor solution. I'll think on it.

Reply:
I don't have a better name. But other than that, this is ready